libcontainer/process_linux.go

```diff
@@ -163,13 +163,53 @@ type setnsProcess struct {
 	initProcessPid  int
 }
 
+// Starts setns process with specified initial CPU affinity.
+func (p *setnsProcess) startWithCPUAffinity() error {
+	aff := p.config.CPUAffinity
+	if aff == nil || aff.Initial == nil {
```

Comment:
Maybe also need to add ` || aff.Count()==0`?

Reply:
I can't think of an input string which will result in cpuset with all zeroes.

Reply:
I didn’t test it, but from a quick view of the code, a special case may be ‘,’. Only a single comma.

Reply:
You are right, and this should be an error. Modified the code and the tests accordingly.

Reply:
Grr, I was in review mode and my comments didn't post.

Yeah, I wanted to say that in that case we would want to return an error rather that ignoring it.

Reply:
@cyphar yes, it returns an error now in case no CPUs are set (and the string is not empty).